pom.xml

```diff
@@ -470,6 +470,15 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.0</version>
+          <configuration>
+             <compilerArgs>
+               <compilerArg>-Werror</compilerArg>
```

Comment:
If I understand it correctly now we generate warnings for "deprecated" and "unchecked" code snippets which will eventually become errors.

Reply:
yes, this is the core part of this change